wp-content/themes/core/assets/css/src/theme/icons/icons.pcss

```diff
@@ -9,10 +9,10 @@
 @font-face {
 	font-family: 'core-icons';
 	src:
-		resolve('assets/fonts/theme/icons-core/core-icons.woff2?k9lxjz') format('woff2'),
-		resolve('assets/fonts/theme/icons-core/core-icons.ttf?k9lxjz') format('truetype'),
-		resolve('assets/fonts/theme/icons-core/core-icons.woff?k9lxjz') format('woff'),
-		resolve('assets/fonts/theme/icons-core/core-icons.svg?k9lxjz#core-icons') format('svg');
+		resolve('wp-content/themes/core/assets/fonts/theme/icons-core/core-icons.woff2?k9lxjz') format('woff2'),
```

Comment:
With the `postcss-assets` changes above, you should be able to revert this, I think.

Reply:
Done!